### PR TITLE
Refactor mt base admin

### PR DESCRIPTION
**Pull request by cchatham**

### CODE REVIEW

File: controllers/base_controller.js [hunk 1]
```diff
@@ -180,17 +184,22 @@ module.exports = function BaseControllerModule(pb) {
...
-        this.siteService = new pb.SiteService();
-        this.siteService.getByUid(this.site, function (err, siteInfo) {
+        var siteService = new SiteService();
+        siteService.getByUid(this.site, function (err, siteInfo) {
+            if(pb.util.isError(err)) {
+                self.reqHandler.serve404();
+                return;
+            }
             self.siteObj = siteInfo;
+            self.siteName = SiteService.isGlobal(siteInfo.uid) ? siteInfo.uid : siteInfo.displayName;
```
Comment (andparker) [inline controllers/base_controller.js]: Do we want the site name for global to be the uid?

Comment (cchatham) [inline controllers/base_controller.js]: Yes because the uid for global is global. This is how we were doing things prior to my change.

**Comment by andparker** [comment]
I am getting a server error when I try to login into the admin console. Are you having this issue?

**Comment by andparker** [comment]
Looks good now Christina.

**Pull request merged**